fix(crons): Fix instructions for using celery beat (#9474)
getsentry/sentry-docs: docs/platforms/python/integrations/celery/crons.mdx

}
```

Next, we need to initialize Sentry. Where to do this depends on how you run beat:

> - If beat is running in your worker process (i.e., you're running your worker with the `-B`/`--beat` option), initialize Sentry in either the `celeryd_init` or `beat_init` signal.

Reviewer: ```suggestion
- If beat is running in your worker process (that is, you're running your worker with the `-B`/`--beat` option), initialize Sentry in either the `celeryd_init` or `beat_init` signal.
```
We try not to use latin abbreviations in the docs
